### DISCUSSION TITLE
Enable the Hypothesis plugin to generate a constrained float with the multiple_of argument specified

### DISCUSSION THREAD

**Discussion by tobi-lipede-oodle** [pydantic/pydantic#2443] [Feature Request]
Currently if we try to generate a model like:

```py
from hypothesis import strategies as st
from pydantic import confloat, BaseModel

class Something(BaseModel):
    field: confloat(ge=5, le=50, multiple_of=4.2)

st.builds(Something).example()
```
Hypothesis will not take the `multiple_of` argument into account. Could the project be modified to do this? As far as I can tell it should just require a small-ish addition to the `resolve_confloat` method.

**Comment by samuelcolvin** [comment]
fixed in #2442
